hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelinePlacementPolicy.java

```diff
@@ -414,6 +425,14 @@ protected DatanodeDetails chooseNodeFromNetworkTopology(
     Node pick = networkTopology.chooseRandom(
         anchor.getNetworkLocation(), excluded);
     DatanodeDetails pickedNode = (DatanodeDetails) pick;
+    if (pickedNode == null) {
+      if (LOG.isDebugEnabled()) {
```

Comment:
Again, I don't think we need the `if (LOG.isDebugEnabled()) {` wrapping the debug log.

Reply:
Updated.